core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/neo/BatchInput.scala

```diff
@@ -0,0 +1,27 @@
+package edu.uci.ics.amber.engine.architecture.worker.neo
+
+import java.util.concurrent.LinkedBlockingDeque
+
+import edu.uci.ics.amber.engine.common.ambertag.LayerTag
+import edu.uci.ics.amber.engine.common.tuple.ITuple
+
+import scala.collection.mutable
+
+class BatchInput{
```

Comment:
We need to make names so that they correspond to architecture components. Rename this to `WorkerInternalQueue`